src/zkevm_specs/evm/typing.py

```diff
@@ -744,6 +744,8 @@ class ExpCircuit:
     rows: List[ExpCircuitRow]
     pad_rows: List[ExpCircuitRow]
 
+    MAX_EXP_STEPS = 100
```

Comment:
I think its better to make this configurable with a fallback default value? Because for a large exponentiation operation, i.e. very large `exponent`, the steps might cross 100.

Reply:
What max row size do you propose for exp_circuit_test?